README.md

```diff
@@ -1,2 +1,3 @@
 # hello-world
-# Everything is safe.
+
+this is frist pull/request.....
```

Comment:
Good point...
I will close this request then. 

Thank you for your time and care. Please do not spend your time on my messy code or life.
Stay blessed.